DigitalLearningSolutions.Web/Controllers/TrackingSystem/CentreConfiguration/RegistrationPromptsController.cs

```diff
@@ -103,88 +112,137 @@ public IActionResult AddRegistrationPromptSelectPrompt(AddRegistrationPromptSele
                 return View(model);
             }
 
+            UpdateTempDataWithSelectPromptModelValues(model);
+
+            return RedirectToAction("AddRegistrationPromptConfigureAnswers");
+        }
+
+        [HttpGet]
+        [Route("Add/ConfigureAnswers")]
+        [ServiceFilter(typeof(RedirectEmptySessionData<AddRegistrationPromptData>))]
+        public IActionResult AddRegistrationPromptConfigureAnswers()
+        {
             var data = TempData.Peek<AddRegistrationPromptData>()!;
-            data.SelectPromptViewModel = model;
-            TempData.Set(data);
+            var viewModel = data.ConfigureAnswersViewModel;
 
-            // TODO: HEEDLS-453 - redirect to next page
-            return RedirectToAction("Index");
+            return View(viewModel);
+        }
+
+        [HttpPost]
+        [Route("Add/ConfigureAnswers")]
+        [ServiceFilter(typeof(RedirectEmptySessionData<AddRegistrationPromptData>))]
+        public IActionResult AddRegistrationPromptConfigureAnswers
+            (RegistrationPromptAnswersViewModel model, string action)

```

Comment:
This is weird formatting - could we set the formatter to put both params onto new lines in this case?

Reply:
This seems to have been caused by slightly conflicting rules. Changed one so default wrapping should now be
```
public void ReallyLongMethodNameWithParameters(
  string param1,
  string param2,
)

Reply:
Perfect, thank you! 👍